### ISSUE TITLE
[Feature]: UI Button to suspend namespace

### ISSUE THREAD

**Issue by krzwiatrzyk** [govirtuo/kube-ns-suspender#79]
Right now you can manually suspend namespace on demand based on changing the annotation.
It would be nice to have an option to do this from UI as it  is easy to see which namespaces are suspended and which are not.

**Comment by krzwiatrzyk** [comment]
Oh and I think i misunderstood the docs. Changing the annotation won't suspend namespace (resources weren't stopped in my case). It only affects displaying namespace state in UI.

**Comment by eze-kiel** [comment]
Can you share your namespaces manifests, please?

**Comment by krzwiatrzyk** [comment]
```
apiVersion: v1
kind: Namespace
metadata:
  annotations:
    kube-ns-suspender/controllerName: kube-ns-suspender
    kube-ns-suspender/desiredState: Running
  creationTimestamp: "2022-05-06T07:44:08Z"
  labels:
    kubernetes.io/metadata.name: test
  name: test
  resourceVersion: "1496"
  uid: e8023a7e-1eae-45a9-a531-17aac645e8f9
spec:
  finalizers:
  - kubernetes
status:
  phase: Active
```

I haven't used annotations for time suspension as a usecase I try to test ATM is as follows:
_I have a cluster with small amount of resources, but I wan't to have access to couple applications without a need of uninstalling them (anre reinstalling + configuring later when needed again) when resources are coming low_
I think that `kube-ns-suspeneder` is an awesome tool for that if I can upscale/downscale namespaces (1 app = 1 namespace) on demand.

**Comment by eze-kiel** [comment]
We are planning to release our `kubectl` plugin that allows a user to suspend and unsuspend namespaces from the CLI.

(example: `kubectl ns [suspend|unsuspend] <namespace>`)

Might this tool suits your needs?

**Comment by krzwiatrzyk** [comment]
YES! It would be awesome (and I believe not only for me). 

But I'm not keen to agree with `ns` as a plugin name because `ns` is a shortcut for namespaces.
I mean you can use `kubectl get ns` to view namespaces. Thus I believe using `ns` name will be misleading

**Comment by eze-kiel** [comment]
Yeah I agree with you, it was an example (internally, the plugin has another name). We'll try to make this plugin public as soon as possible, I'll keep you posted!

**Comment by eze-kiel** [comment]
well, as you're not the first one to ask for this feature, I'll reopen the issue and try to work on it in the next days.

**Comment by eze-kiel** [comment]
Hi @Brice187, a quick update to inform you that the new web UI has been developed and is currently being tested, it should be released before the end of the day as version `v2.1.0`

Cc @krzwiatrzyk as you might be interested!

**Comment by Brice187** [comment]
Please push the v2.1.0 image to the registry ;)

**Comment by eze-kiel** [comment]
The PR is still open and being reviewed :wink:
